source/retryable-writes/tests/insertOne-serverErrors.yml

```diff
@@ -262,6 +262,28 @@ tests:
                     - { _id: 1, x: 11 }
                     - { _id: 2, x: 22 }
                     - { _id: 3, x: 33 }
+    -
+        description: "InsertOne succeeds after ExceededTimeLimit"
+        failPoint:
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+                failCommands: ["insert"]
+                writeConcernError:
```

Comment:
Is ExceededTimeLimit a write concern error or a general write error? If the latter, we should use `errorCode` instead of `writeConcernError.code`.

[Determining retryable errors](https://github.com/mongodb/specifications/blob/master/source/retryable-writes/retryable-writes.rst#determining-retryable-errors) doesn't really care since we'll look for codes in either place, but it'd be good to have this example be consistent with what the server might actually return.

Reply:
Yes 😓 Thanks for catching that.